README.md

```diff
@@ -90,12 +98,34 @@ deploystack uninstall
 ```
 
 The above command will delete the associated resources so there will be no billable charges made afterwards.
+In order to remove all of the resources, you will need to delete the Logging Sink that it creates at an organisational level. To do this, please complete the following steps:
+
+* Navigate to *Cloud Logging*
+* Navigate to *Logs Router*
+* In the menu at the top, please select your organisation
+* Delete the entry of type BigQuery dataset named audit-log-org-sink
 
 Note: This will also destroy the BigQuery dataset as the following option in `main.tf` is set to `true`: `delete_contents_on_destroy`.
 
-### Customizing to your environment
+### Customizing to Your Environment
+
+In this section you will find different ways to customise the architecture to your needs.
+
+#### Example Log Filters
+
+**GWS Audit Log Filter Example**
```

Comment:
Could we elaborate a bit on what these logs are? (for example, explain what kind of logs are system events etc...)